1.9.0

```diff
@@ -228,6 +228,25 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Lead",
+      "instructionTypeName": "mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-58367",
+          "description": "Create new Lead List View",
+          "actionType": "ListView",
+          "actionTypeName": "createListView",
+          "optionsMap": {
+            "fullName": "Lead.TodaysLeads",
```

Comment:
use the actionTypeName here instead of fullname, otherwise its working

Reply:
done